### ISSUE TITLE
Using `consult-dir` in minibuffer requires `enable-recursive-minibuffers` non-nil

### ISSUE THREAD

**Issue by dalanicolai** [karthink/consult-dir#28]
As mentioned in the title, using `consult-dir` from the minibuffer, requires the variable of `enable-recursive-minibuffers` set to t.
I could not find this being documented. Otherwise, it probably would be nice if `consult-dir` sets this automatically.

**Comment by karthink** [comment]
@dalanicolai To use consult-dir, the user is not required to set, or even be aware of the variable `enable-recursive-minibuffers`. `consult-dir` enables recursive minibuffers if needed, and restores the prior state when exiting.

Does `consult-dir` not work for you from the minibuffer if you `(setq enable-recursive-minibuffers nil)`?

**Comment by dalanicolai** [comment]
Indeed, I simply wanted to test the package, but when trying to invoke `consult-dir` from the minibuffer, a message appeared that I was 'trying to invoke a command from the minibuffer'. After setting `enable-recursive-minibuffers` to t things worked fine.

**Comment by karthink** [comment]
@dalanicolai Could you test it again, by first setting `enable-recursive-minibuffers` to `nil`? `consult-dir` automatically sets it to `t` when invoked from the minibuffer, so this shouldn't be happening.
